sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs

```diff
@@ -1505,21 +1505,21 @@ public virtual Response AddFeedback(string projectName, RequestContent content,
         /// </code>
         /// 
         /// </remarks>
-        public virtual AsyncPageable<BinaryData> GetProjectsAsync(int? top = null, int? skip = null, int? maxpagesize = null, RequestContext context = null)
+        public virtual AsyncPageable<BinaryData> GetProjectsAsync(int? maxCount = null, int? skip = null, int? maxpagesize = null, RequestContext context = null)
```

Comment:
This is a breaking SDK API change. We'd need some sort of customization to skip doing this for GA'd DPGs.

@tg-msft I also found that neither the General or .NET guidelines have anything on what we want these parameters/properties/etc to be. I know you've mentioned it in reviews in the past, but could this get written down in guidelines so we have a clear picture of what's expected?

Reply:
@heaths Thanks. I'm going to add an option `disable-pagination-top-renaming` in the config. Its default value is `false` (since I believe we want the renaming as the default behavior for upcoming SDKs), and for device update I'll change the value to `true`.

Reply:
I've added `disable-pagination-top-renaming: true` for QuestionAnswering. See the latest change.